groups/promo-infrastructure/profiles/shared-services-eu-west-2/vars

```diff
@@ -0,0 +1,17 @@
+aws_profile = "shared-services-eu-west-2"
+aws_region  = "eu-west-2"
+aws_account = "shared-services"
+
+# shorthand
+account = "shared"
+region  = "euw2"
+
+# Application details
+application = "promo"
+environment = "live"
```

Comment:
should be shared-services

Reply:
As discussed this has been renamed to promo-proxy